### ISSUE TITLE
Allow specification of artifacts to load on MicrocksContainer creation

### ISSUE THREAD

**Issue by lbroudoux** [microcks/microcks-testcontainers-go#11]
### Reason/Context

As of today, loading artifacts into the Microcks container should be done after the container is started-up using 2 methods:
* `importAsMainArtifact(ctx, artifact)` or,
* `importAsSecondaryArtifact(ctx, artifact)`

However, this way of doing things doesn't really fit with existing modules' logic - all of them have `WithSomething()` fluent API style methods that allow config definition at creation. Moreover, our approach can sometimes be harder to achieve cause it means you have to retrieve the container once initialized (depending on the test framework, this can be tricky) to import artifacts.

### Description

Ideally, we'd like to be able to have something like:

```go
microcksContainer, err := microcks.RunContainer(ctx, 
   testcontainers.WithImage("quay.io/microcks/microcks-uber:nightly"),
   testcontainers.WithPrimaryArtifact("testdata/apipastries-openapi.yaml"))
```

### Implementation ideas

See if implementation ideas from https://github.com/microcks/microcks-testcontainers-java/issues/26 can be reused/adapted here to Go

**Comment by JulienBreux** [comment]
Note: I replace Primary by Main to push strong consistency 😉

**Comment by lbroudoux** [comment]
Now solved with #14. Thanks to @JulienBreux!
